### ISSUE TITLE
trezor suite app issue: asset not loaded correctly!

### ISSUE THREAD

**Issue by BenGe1** [trezor/trezor-suite#8666]
**Describe the bug**
When hardware wallet is connected with  (Mac) desktop suite APP,  It takes a while (a few minutes) to load the wallet balance and then failed and indicates “ asset not loaded correctly” 

**Info:**

-   Remembered Wallet: yes
-   Suite Version: app
-  I had no such problem with trezor suites app until recently( after updating app version?). 

Plz noted: 1. It works smoothly in trezor suite WEB version.   2. I have to use VPN to access trezor website in my country.  3. Both my desktop application and firmware are most updated on the same computer.  4. I have two Trezor hardware wallets, they have encountered same issue in both my computers.

I have tried all the tips ( VPN on/off, reset app, reset computer, tor on/off, reinstall suite app, turn off virus scan app..)  it’s just not working.

**Expected behavior**
I highly suspect this issue is related to the update of the desktop app. Please review and fix it.

**Comment by sime** [comment]
@BenGe1 Are you using a proxy? Seems related to this: https://github.com/trezor/trezor-suite/issues/6312

**Comment by BenGe1** [comment]
hi plz read this context info so that u can save some time and give me the accurate instructions.

I have tried it with and without VPN, not work.

This is very frustrating and wasted a lot of my time. I had no such problem with trezor suites app until recently, which is still ongoing.

Plz noted: 1. It works smoothly in trezor suite webpage.   2. I have to use VPN to access trezor website in my country.  3. Both my desktop application and firmware are most updated on the same computer.

So I’m very curious what is going on with the desktop APP?  I have tried all the tips ( VPN on/off, reset app, reset computer, tor on/off)  it’s just not working. It is connected and then takes a while to load the page and then indicates “ asset not loaded correctly”

 I have two trezor hardware wallets, this issue is happening to both of them on all my computers(Mac and MacBook) . So I think this is a problem with the desktop suite APP


发自我的 iPhone

在 2023年6月12日，19:38，Simon Males ***@***.***> 写道：

﻿

@BenGe1<https://github.com/BenGe1> Are you using a proxy? Seems related to this: #6312<https://github.com/trezor/trezor-suite/issues/6312>

**Comment by sime** [comment]
@BenGe1 Do you have any system proxies setup?


![Image](https://github.com/trezor/trezor-suite/assets/216917/d5ab5eea-2a07-42a9-91ba-67e9b3bc4653)

**Comment by BenGe1** [comment]
hi Sime, just checked, when I logged in VPN (clashX pro), this system proxies are automatically on.  I tried Trezor suite again with the proxies off and on, but the issue stays in both cases. 
The thing is, without VPN, I can not even access trezor.io,  I'm not sure if this affects logging in suite APP...
thanks.

**Comment by BenGe1** [comment]
> @BenGe1 Do you have any system proxies setup?
> 
> ![Image](https://user-images.githubusercontent.com/216917/245129611-d5ab5eea-2a07-42a9-91ba-67e9b3bc4653.png)

hi sime, u're right! It is related to system proxies, what I just did was I used another cheap VPN, which doesn't initiate system proxy while working.  and then I can log in suite APP!
so the logic here is,  I have to use a VPN to access trezor.io and trezor suite, but it has to leave the system proxy off when VPN is on. ONLY with those two conditions, I can log in suite APP here in China. 

I hope this can help other users in China.  meanwhile ,  hopefully Trezor team can solve this issue? some good vpn does automatically turns system proxies on. 

thanks again Sime!
